apps/desktop/src/components/v3/StackCard.svelte

```diff
@@ -20,12 +19,6 @@
 </script>
 
 <div class="stack-card text-15 text-bold text-" class:selected>
-	<div class="header">
-		<BranchLabel name={stackName} readonly />
-		<div>
-			{@render contextMenu()}
-		</div>
-	</div>
```

Comment:
Let's not drop the context menu? It's the only way to unapply a stack.

Reply:
Oh yeah, forgot about that. Maybe we could invent a context menu that looks nice in a "headless" mode (without stack name)